### ISSUE TITLE
Ability to run jobs in serial mode

### ISSUE THREAD

**Issue by andreoid** [gocd/gocd#268]
Hi,

Sometimes I wish to run my jobs in serial, one after the after, rather than in parallel.

My example:

I have jobs A, B and C in one of my Stages. Each job connects to a service and I get conflicts if they all run at the same time.

I don't want to have them as separate pipeline stages, they are conceptually the same stage.

I don't want them as tasks in the same job, as they each produce their own test results etc.

I need them to run one after the other. I would suggest that in the Job Settings menu an On Success flag is added, much like you have for the Stage Settings menu.

I currently get around this, by tying them to all run on a single agent, using a named resource - but that isn't ideal, as it causes build queues if the agent is busy.

**Comment by arvindsv** [comment]
Closing in favor of #827.
